### PR TITLE
new package: libfcl

### PR DESCRIPTION
**Pull request by pvonmoradi**
Adds [fcl](https://github.com/flexible-collision-library/fcl), a dependency of DART.
Depends on [libccd](https://github.com/termux/termux-packages/pull/11480/) and [octomap](https://github.com/termux/termux-packages/pull/11486)

### CODE REVIEW

File: packages/libfcl/build.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+TERMUX_PKG_GROUPS="science"
+TERMUX_PKG_LICENSE="BSD 3-Clause"
+TERMUX_PKG_MAINTAINER="Pooya Moradi <pvonmoradi@gmail.com>"
+TERMUX_PKG_VERSION="0.7.0"
+TERMUX_PKG_SRCURL="https://github.com/flexible-collision-library/fcl/archive/refs/tags/$TERMUX_PKG_VERSION.tar.gz"
+TERMUX_PKG_SHA256=90409e940b24045987506a6b239424a4222e2daf648c86dd146cbcb692ebdcbc
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_FORCE_CMAKE=true
+TERMUX_PKG_BUILD_DEPENDS="eigen, libccd, octomap"
```
Comment (Grimler91) [inline packages/libfcl/build.sh:12]: libccd and octomap are runtime dependencies (libfcl.so are linked to them) so they need to be in `TERMUX_PKG_DEPENDS`. Are you sure eigen is needed? Build doesn't mention eigen ever so I am not sure it is used for anything

File: packages/libfcl/build.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS='
```
Comment (Grimler91) [inline packages/libfcl/build.sh:14]: Please switch to double quotes "

File: packages/libfcl/build.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/flexible-collision-library/fcl"
+TERMUX_PKG_DESCRIPTION="flexible collision library"
```
Comment (Grimler91) [inline packages/libfcl/build.sh:2]: Please start with capital letter, i.e. Flexible

**Review by Grimler91 — CHANGES_REQUESTED** [review]
Thanks for the PR! See comments

File: packages/libfcl/build.sh (hunk 1 above)
Comment (pvonmoradi) [reply packages/libfcl/build.sh:12]: Yes, It is [declared in its CMakeLists](https://github.com/flexible-collision-library/fcl/blob/master/CMakeLists.txt#L171)

**Comment by pvonmoradi** [comment]
@Grimler91 Rebased to master but libm.so problem still persists

**Comment by Grimler91** [comment]
Hmmm, seems android-r25-api-24-v1 is hardcoded or referenced somewhere. I'll have a look

**Comment by Grimler91** [comment]
Caused by libccd, should work with added commits

**Pull request merged**